### PR TITLE
Fix installation instructions to download correct file, remove zip files from build step

### PR DESCRIPTION
**Pull request by olafghanizadeh**
Currently, installation instructions tells you to download the .zip file from releases and use that to install in LocalWP, however, this is not supported. Only .tar.gz is supported.

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -23,10 +23,7 @@ jobs:
 
             - name: Build JS
               run: npm run build
-
-            - name: Zip files
-              run: zip release.zip ./* -r
-
+              
             - name: Tar files
               run: tar --exclude='release.zip' -czf release.tar.gz *
```
Comment (JorgenStenshaugen) [inline .github/workflows/release.yml:28]: Not important but you could also remove the exclude, since it's shouldn't be needed anymore

**Review by JorgenStenshaugen — CHANGES_REQUESTED** [review]
Thanks for the contribution! Just a minor request to remove the exclude zip line from the workflow

File: .github/workflows/release.yml (hunk 1 above)
Comment (olafghanizadeh) [reply .github/workflows/release.yml]: Fixed and squashed with the relevant commit :)

**Review by JorgenStenshaugen — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**